### PR TITLE
Misc changes at the colour tagging feature 

### PR DESCRIPTION
**Pull request by schlenkibus**
moved colour line in info
changed css to be more centred horizontal
changed highlight
added automatic attribute setting for colour if not present (server proxy now has "setPresetAttributeHidden", that starts cuckoo transaction)

### CODE REVIEW

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java [hunk 1]
```diff
@@ -722,7 +722,20 @@ public void setMacroControlInfo(int parameterID, String text) {
...
+			return;
+		
+		StaticURI.Path path = new StaticURI.Path("presets", "banks", "set-preset-attribute-hidden");
+		StaticURI uri = new StaticURI(path, new StaticURI.KeyValue("uuid", thePreset.getUUID()), new StaticURI.KeyValue("key", key),
+				new StaticURI.KeyValue("value", value));
+		queueJob(uri, false);
+	}
+	
 	public void setPresetAttribute(Preset thePreset, String key, String value) {
+		if(thePreset == null)
```
Comment (hhoegelo) [inline NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java]: I think, if the method is called with thePreset == null, there is a bug on callers side which should be fixed.

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java [hunk 2]
```diff
@@ -722,7 +722,20 @@ public void setMacroControlInfo(int parameterID, String text) {
 		queueJob(uri, false);
 	}
 
+	public void setPresetAttributeHidden(Preset thePreset, String key, String value) {
```
Comment (hhoegelo) [inline NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java]: I don't believe that we need such an ugly function - as far as I see, all you want to achieve is defaulting empty color attribute to 'none' - this should better be done on playgrounds side - for example in Preset CTor.

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java (hunk 1 above)
Comment (schlenkibus) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java]: public void setPresetAttribute(Preset thePreset, String key, String value) {
		if(thePreset == null)
			return;

Comment (schlenkibus) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java]: Or do you mean that shouldn't be there?

Comment (hhoegelo) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/ServerProxy.java]: shouldn't

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/PresetInfoDialog.java [hunk 3]
```diff
@@ -360,9 +358,14 @@ private void updateInfo(Preset preset) {
 			Bank bank = preset.getParent();
 			bankName.setText(bank.getOrderNumber() + " - " + bank.getTitleName());
 			
-			ColorTag.Color c = ColorTag.Color.valueOf(preset.getAttribute("color"));
-			updateCurrentHighlight(colors[c.ordinal()]);
 			
+			String presetColorString = preset.getAttribute("color");
```
Comment (hhoegelo) [inline NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/PresetInfoDialog.java]: hier wuerde ich die sonderbehandlung des Strings "" in ColorTag.Color.valueOf verschieben

File: playground/src/presets/BankActions.cpp [hunk 4]
```diff
@@ -788,6 +788,20 @@ BankActions::BankActions(PresetManager &presetManager) :
     }
   });
 
+  addAction("set-preset-attribute-hidden", [&] (shared_ptr<NetworkRequest> request) mutable
```
Comment (hhoegelo) [inline playground/src/presets/BankActions.cpp]: das kann doch nun weg, oder?

**Pull request merged**